src/vuex/modules/connection.js

```diff
@@ -17,11 +18,24 @@ export default function() {
   }
 
   const actions = {
-    async checkForPersistedNetwork({ commit }) {
-      const network = localStorage.getItem(`network`)
-      if (network) {
-        await commit(`setNetworkId`, JSON.parse(network))
+    async checkForPersistedNetwork({ dispatch, commit }) {
+      const network = JSON.parse(localStorage.getItem(`network`))
+      const { data } = await apollo.query({
+        query: Networks
+      })
+      let availNetworks = Object.values(data.networks).map( network => network.id)
+      if (network && availNetworks.includes(network)) {
+        await commit(`setNetworkId`, network)
+      } else if (network && !availNetworks.includes(network)) {
```

Comment:
```suggestion
      } else {
```

Reply:
Makes sense now that you put it that way :stuck_out_tongue_winking_eye: